split images and added link to current user-stories

dev-smart-ui opened the pull request:
resolves #278

dennyglee commented:
Thanks @dev-smart-ui - this is looking pretty good.  Saying this, could you add some more vertical spacing between the different logos.  The horizontal spacing is quite good but the vertical spacing seems squat.

As well, could you include all of the logos from the original diagram as well as any new ones?  Thanks!

dev-smart-ui commented:
Hey @dennyglee ! 
Thanks for your feddback. 

I added more row spacing between the icons. As for quontity of icons I added all of them from the current scheme (included Kubit), but I changed the grid from 6(columns) x 4(rows) to the 5(columns) x 5(rows). In other way we will get only one icons for last row. It looks not well. 

What do you think?

dennyglee commented on src/components/pages/community/ContributeToDeltaSection/contributors.js:
```diff
@@ -0,0 +1,153 @@
+import twilio from "./img/twilio.png";
+import ucDavis from "./img/ucDavis.png";
+
+export const contributors = [
```
Could we rename this to `orgContributors` vs `contributors` so we don't confuse between organizations vs. contributors (maintainers).

dennyglee replied:
thanks!

dennyglee commented on src/components/pages/community/ContributeToDeltaSection/index.jsx:
```diff
@@ -28,10 +45,18 @@ const ContributeToDeltaSection = () => (
-      <ContributeToDeltaColumn>
-        <img src={organizationsImg} alt="" />
-      </ContributeToDeltaColumn>
+    <Grid columns={{ xs: 2, sm: 3, md: 4, lg: 5 }} gutter="lg" rowGutter="xl">
```
Can we try rowGutter = "xxl"?

dennyglee replied:
thanks, looks great!

dennyglee reviewed (changes requested):
Two quick changes - thanks!

dennyglee commented:
Thanks @dev-smart-ui  - can we try xxl row gutter just to see if it looks better?  As for the icons - that's fair, just wanted to confirm that I can add another 5 icons :)

dennyglee reviewed (approved):
awesome, thanks!

dennyglee reviewed (approved):
awesome, thanks!
